easybuild/easyconfigs/x/XCfun/XCfun-20180122-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MakeCp'
+
+name = 'XCfun'
+version = '20180122'
+commit = 'acd11bfbdcc2d2fb9c100111545bd479e6e2c452'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://dftlibs.org/xcfun/'
+description = """ XCFun is a library of DFT exchange-correlation (XC) functionals.
+ It is based on automatic differentiation and can therefore generate arbitrary order
+ derivatives of these functionals. """
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/dftlibs/xcfun/archive/']
+sources = ['%s.tar.gz' % commit]
+
+patches = ['%(name)s-%(version)s-shared_library-intel.patch']
+
+checksums = [
+    # acd11bfbdcc2d2fb9c100111545bd479e6e2c452.tar.gz
+    '1dd1bac10fd12682ac0b65c51fd1fd5928104d1210165ef7d9e433ec99090e4e',
+    # XCfun-20180122-shared_library-intel.patch
+    '4386590cffcae74e0ee443d2cbff8e927a2de026a5e0842b7d268a9f4f490054',
+]
+
+
+dependencies = [
+    ('Python', '2.7.14'),
+]
+
+prebuildopts = 'cp Makefile.intel Makefile && '
+
+files_to_copy = ['lib', 'include', 'doc', 'test']
+
+sanity_check_paths = {
+    'files': ['lib/libxcfun.%s' % x for x in ['a', SHLIB_EXT]] + ['include/xcfun.h'],
```

Comment:
using a list comprehension here is a bit overkill, better 'unroll' it (and sort alphabetically):

```python
'files': ['include/xcfun.h', 'lib/libxcfun.a', 'lib/libxcfun.%s' % SHLIB_EXT],